### PR TITLE
Use a forked OpusCleaner in `release`

### PR DESCRIPTION
**Pull request by gregtatum**
This adds support for bs, id, sr, tr, vi. This should not be merged into main, as I'd prefer to add support upstream if I can.

### CODE REVIEW

File: pipeline/clean/requirements/clean.in [hunk 1]
```diff
@@ -1,4 +1,5 @@
-opuscleaner==0.4.0
+# Add support for bs, id, sr, tr, vi.
+git+https://github.com/gregtatum/OpusCleaner.git@spring2024
```
Comment (eu9ene) [inline pipeline/clean/requirements/clean.in:2]: Even as a temporary solution, I would prefer to merge it to our Mozilla fork https://github.com/mozilla/OpusCleaner and use it from there.

Comment (eu9ene) [inline pipeline/clean/requirements/clean.in:2]: It's worth adding a TODO to switch to the main repo when it's merged.

Comment (gregtatum) [reply pipeline/clean/requirements/clean.in:2]: How do I get access to this repo? Can I directly push to it?

Comment (eu9ene) [reply pipeline/clean/requirements/clean.in:2]: I added our team there and also at https://github.com/mozilla/OpusTrainer. So feel free to sync it and PR there.

Comment (eu9ene) [reply pipeline/clean/requirements/clean.in:2]: Or directly push, up to you

Comment (gregtatum) [reply pipeline/clean/requirements/clean.in]: I directly pushed to spring2024.

**Comment by gregtatum** [comment]
```
[task 2024-06-20T20:36:08.985Z] ERROR: Can't verify hashes for these requirements because we don't have a way to hash version control repositories:
[task 2024-06-20T20:36:08.985Z]     opuscleaner@ git+https://github.com/mozilla/OpusCleaner.git@spring2024 from git+https://github.com/mozilla/OpusCleaner.git@spring2024 (from -r /builds/worker/checkouts/vcs/pipeline/clean/requirements/clean.txt (line 187))
```

**Review by eu9ene — APPROVED** [review]

**Pull request merged**